### PR TITLE
Upgrade testfloat to 3e version.

### PR DESCRIPTION
**Pull request by zongbox**
Upgrade TestFloat to 3e version.  It had got more test case from 3a version to 3e version.

### CODE REVIEW

File: meta-oe/recipes-test/testfloat/testfloat_3e.bb [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 
 SRC_URI = "\
-    http://www.jhauser.us/arithmetic/TestFloat-3a.zip;name=TestFloat \
-    http://www.jhauser.us/arithmetic/SoftFloat-3a.zip;name=SoftFloat \
-    file://0001-Makefile-for-cross-compile-SoftFloat.patch \
-    file://0002-Makefile-for-cross-compile-TestFloat.patch \
+    http://www.jhauser.us/arithmetic/TestFloat-3e.zip;name=TestFloat \
+    http://www.jhauser.us/arithmetic/SoftFloat-3e.zip;name=SoftFloat \
+    file://0001-Makefile-for-cross-compile-SoftFloat-3e.patch \
+    file://0002-Makefile-for-cross-compile-TestFloat-3e.patch \
```
Comment (kraj) [inline meta-oe/recipes-test/testfloat/testfloat_3e.bb:15]: the patch names perhaps should not contain `-3e` in name 

we get errors like

```
WARNING: /mnt/b/yoe/master/sources/meta-openembedded/meta-oe/recipes-test/testfloat/testfloat_3e.bb: Unable to get checksum fo
r testfloat SRC_URI entry 0001-Makefile-for-cross-compile-SoftFloat-3e.patch: file could not be found
WARNING: /mnt/b/yoe/master/sources/meta-openembedded/meta-oe/recipes-test/testfloat/testfloat_3e.bb: Unable to get checksum fo
r testfloat SRC_URI entry 0002-Makefile-for-cross-compile-TestFloat-3e.patch: file could not be found
```

**Comment by zongbox** [comment]
Khem Raj <notifications@github.com> 於 2020年5月7日 週四 下午2:19寫道：

> *@kraj* requested changes on this pull request.
> ------------------------------
>
> In meta-oe/recipes-test/testfloat/testfloat_3e.bb
> <https://github.com/openembedded/meta-openembedded/pull/221#discussion_r421264163>
> :
>
> >
>  SRC_URI = "\
> -    http://www.jhauser.us/arithmetic/TestFloat-3a.zip;name=TestFloat \
> -    http://www.jhauser.us/arithmetic/SoftFloat-3a.zip;name=SoftFloat \
> -    file://0001-Makefile-for-cross-compile-SoftFloat.patch \
> -    file://0002-Makefile-for-cross-compile-TestFloat.patch \
> +    http://www.jhauser.us/arithmetic/TestFloat-3e.zip;name=TestFloat \
> +    http://www.jhauser.us/arithmetic/SoftFloat-3e.zip;name=SoftFloat \
> +    file://0001-Makefile-for-cross-compile-SoftFloat-3e.patch \
> +    file://0002-Makefile-for-cross-compile-TestFloat-3e.patch \
>
> the patch names perhaps should not contain -3e in name
>
> we get errors like
>
> WARNING: /mnt/b/yoe/master/sources/meta-openembedded/meta-oe/recipes-test/testfloat/testfloat_3e.bb: Unable to get checksum fo
> r testfloat SRC_URI entry 0001-Makefile-for-cross-compile-SoftFloat-3e.patch: file could not be found
> WARNING: /mnt/b/yoe/master/sources/meta-openembedded/meta-oe/recipes-test/testfloat/testfloat_3e.bb: Unable to get checksum fo
> r testfloat SRC_URI entry 0002-Makefile-for-cross-compile-TestFloat-3e.patch: file could not be found
>
> —
>

Sorry, it's embarrassing to that. The '-3e' suffix is used at beginning.
Finally, only remove the suffix of the name of patch files, but forgot
changing them in bb file. Let me send the next version patch to fix it.

**Comment by zongbox** [comment]
@kraj I had force pushed the branch zongbox:testfloat_3e.  Thanks.

**Comment by kraj** [comment]
merged with 630d752a0c

**Pull request closed without merging**